hikari/internal/ux.py

```diff
@@ -198,6 +206,12 @@ def print_banner(package: typing.Optional[str], allow_color: bool, force_color:
         "system_description": " ".join(filtered_system_bits),
     }
 
+    if extra_args:
+        for key in extra_args:
+            if key in args:
+                raise ValueError(f"Cannot overwrite $-substitution `{key}`. Please use a different key.")
```

Comment:
One last thing I just realized I forgot to comment on, we should also document that it raises a value error:

```
Raises
------
builtins.ValueError
    <Hi>
```

Reply:
Should this also be added to `RestBot.print_banner` and `GatewayBot.print_banner`?

Reply:
Yep :)

Reply:
added :)